webapp/src/shared/components/Avatar/Avatar.css

```
@@ -0,0 +1,50 @@
+.avatar-status--online {
+    background-color: var(--color-online);
+}
+
+.avatar-status--offline {
+    background-color: var(--color-offline);
+}
+
+.avatar-status--playing {
+    background: var(--gradient-game);
+}
+
+.avatar-large {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.avatar-small__image-wrapper, .avatar-large__image-wrapper {
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    margin: 0.25rem;
+}
+
+.avatar-large__image {
+    width: 200px;
```

```suggestion
    width: 200px;
    height: 200px;
```
From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img)
> Use both [width](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-width) and [height](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-height) to set the intrinsic size of the image, allowing it to take up space before it loads, to mitigate content layout shifts.

Same for the `.avatar-small__image` class.

---

good catch! reading the documentation we also could to get rid off `px` units: 

> [height](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-height)
> : The intrinsic height of the image, in pixels. Must be an integer without a unit.

I'll remove it

---

I say nothing about `px` it is only for the `img` tag properties itself